### ISSUE TITLE
[4.0] "Highlight Links" destroy layout on pagination

### ISSUE THREAD

**Issue by ghost** [joomla/joomla-cms#34363]
### What needs to be fixed
1. Enable in user profile the accessibility setting `Highlight Links`
2. Open as example Modules: Administrator
3. set visible modules at a page to `5`

User get (for me) unexpected pagination style:

![image](https://user-images.githubusercontent.com/78906980/120479048-c8c45280-c3ad-11eb-8b77-2d11825086a2.png)
![image](https://user-images.githubusercontent.com/78906980/120479078-cfeb6080-c3ad-11eb-8bed-de1fb8b2f88e.png)

### Why this should be fixed
It looks different (page 1 and page 4) and difficult to touch on mobile.

### How would you fix it
No idea.

**Comment by infograf768** [comment]
Looks like this may come from the padding here.


https://github.com/joomla/joomla-cms/blob/7477fd7733058c7003d3b43525dab0e48ddffef7/administrator/templates/atum/scss/blocks/_global.scss#L45-L52

If I take the padding off, I get

<img width="1216" alt="Screen Shot 2021-06-02 at 17 54 50" src="https://user-images.githubusercontent.com/869724/120512464-ad684000-c3cb-11eb-9a94-08504dd7a28f.png">

As this has an effect on the headers, it may be necessary to add a specific css with the .pagination class.

**Comment by infograf768** [comment]
@drmenzelit

**Comment by drmenzelit** [comment]
Please test #34394
